lib/iamvery/type/legacy_duration.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Iamvery.Type.LegacyDuration do
```

Comment:
✍️ I'm not a huge fan of the name "legacy" here. It's really just a "less precise duration". However, I don't care to make that decision right now. Just going to port over as-is.